tests/page-objects/login/login.wdio.page.js

```diff
@@ -29,7 +29,9 @@ const cookieLogin = async (username = auth.username, password = auth.password) =
   const cookieArray = utils.parseCookieResponse(resp.headers['set-cookie']);
 
   await browser.setCookies(cookieArray);
-  await utils.setupUserDoc(username);
+  if (createUser) {
```

Comment:
I believe this is only used for the admin user. Let's just get rid of it here and move it to the `onPrepare` function in the `wdio.conf.js`

Reply:
Done